addons/stock_account/wizard/stock_valuation_layer_revaluation.py

```diff
@@ -55,10 +55,12 @@ def default_get(self, default_fields):
     product_uom_name = fields.Char("Unit", related='product_id.uom_id.name')
     current_value_svl = fields.Float("Current Value", compute='_compute_current_value_svl')
     current_quantity_svl = fields.Float("Current Quantity", compute='_compute_current_value_svl')
+    lot_ids = fields.Many2many('stock.lot', compute='_compute_lot_ids', help="All concerned lot/serial number")
```

Comment:
You should use adjusted_layer_ids instead of context and replace lot_id by lot_ids. Not use both